lib/chef/resource/aws_db_subnet_group.rb

```diff
@@ -0,0 +1,20 @@
+require 'chef/provisioning/aws_driver/aws_resource'
+require 'chef/resource/aws_subnet'
+
+class Chef::Resource::AwsDbSubnetGroup < Chef::Provisioning::AWSDriver::AWSResource
+  aws_sdk_type AWS::RDS
```

Comment:
Unfortunately there isn't a group object in the v1 SDK.  This is just here to make provisioning happy.  I should comment it.